acme/tests/client_test.py

```diff
@@ -5,19 +5,17 @@
 import json
 import unittest
 
-from six.moves import http_client  # pylint: disable=import-error
-
+import OpenSSL
```

Comment:
Style guide says to ignore case: "Within each grouping, imports should be sorted lexicographically, ignoring case, according to each module’s full package path."

https://google.github.io/styleguide/pyguide.html#313-imports-formatting

Looks like this can by achieved by setting `order_by_type` to `False` in addition to the `force_sort_within_sections` to `True` that we're already doing.

https://github.com/timothycrosley/isort/issues/318#issuecomment-142818678

Reply:
Added that. It fixes all related inconsistencies regarding capitalized letters.